account-kit/react/src/metrics.ts

```diff
@@ -0,0 +1,24 @@
+import { createLogger } from "@account-kit/logging";
+import type { AlchemyAccountsUIConfig } from "./types.js";
+import { VERSION } from "./version.js";
+
+export type ReactEventsSchema = readonly [
+  {
```

Comment:
is config_created a dupe of the one in core. I'd recommend we try to reduce the amount of duplicate events as much as possible. It'll make our lives easier when we analyze.

Reply:
This one isn't a dupe exactly, the core config one only includes the core configuration details whereas this one includes the UI config settings. The number of `config_created` where packageName = `react` will be a subset of the total `config_created` events with name = "core"

Reply:
I didn't wanna merge the events mainly because I didn't want to leak the `core` impl into `react` because of metrics

Reply:
Gotcha ok we can keep it if they are distinct different events. But wouldn't they share an anonid so we'd be able to map it correctly? Anyway I'm not gonna block this we can always iterate to something better.

Reply:
Yea they will share anonId so we can merge these events on the backend